articles/imagesharp/pixelbuffers.md

```diff
@@ -61,7 +106,7 @@ image.Mutate(c => c.ProcessPixelRowsAsVector4(row =>
 
 This API receives a @"SixLabors.ImageSharp.Processing.PixelRowOperation" instance as input, and uses it to modify the pixel data of the target image. It does so by automatically executing the input operation in parallel, on multiple pixel rows at the same time, to fully leverage the power of modern multi-core CPUs. The `ProcessPixelRowsAsVector4` extension also takes care of converting the pixel data to/from the `Vector4` format, which means the same operation can be used to easily process images of any existing pixel-format, without having to implement the processing logic again for each of them.
 
-This extension offers the fastest, easiest and most flexible way to implement custom image processors in ImageSharp.
```

Comment:
`ProcessPixelRowsAsVector4` is really not a silver-bullet "the fastest, easiest" method, but one of the many options with it's pros and cons.